src/main/resources/assets/clientcommands/lang/en_us.json

```
@@ -245,6 +245,58 @@
+  "commands.cvillager.help.noClock": "Help: villager RNG manipulation requires a 20 Hz clock",
+  "commands.cvillager.help.tooFast": "Help: villager RNG manipulation requires a 20 Hz clock, please be advised that your clock seems to be running faster",
+  "commands.cvillager.help.tooSlow": "Help: villager RNG manipulation requires a 20 Hz clock, please be advised that your clock seems to be running slower",
+  "commands.cvillager.inSync": "Your villager's RNG is cracked",
+  "commands.cvillager.listGoals.noGoals": "There are no villager goals",
+  "commands.cvillager.listGoals.success": "There are %s villager goals:",
+  "commands.cvillager.needVillagerManipulation": "Villager manipulation is not enabled",
+  "commands.cvillager.noCrackedVillagerPresent": "There was no cracked villager available to use",
+  "commands.cvillager.noProfession": "The targeted villager has no profession",
+  "commands.cvillager.notAVillager": "Target was not a villager",
+  "commands.cvillager.notLevel1": "The targeted villager has level above 1",
+  "commands.cvillager.uncracked": "Your villager's RNG is not cracked (~18 quintillion seeds remain)",
+  "commands.cvillager.partiallyCracked": "Your villager's RNG is partially cracked (~83m seeds remain)",
+  "commands.cvillager.removeGoal.invalidIndex": "Unable to remove goal %s, you only have %s goals",
+  "commands.cvillager.removeGoal.success": "Successfully removed goal %s",
+  "commands.cvillager.resetCracker": "Successfully reset cracked villager RNG",
+  "commands.cvillager.success": "Got the correct trade with correction of %sms",
+  "commands.cvillager.target.cleared": "Target entity cleared",
+  "commands.cvillager.target.set": "Target entity set",
+
+  "villagerManip.reset": "Restarting villager RNG manipulation. Reason: %s",
+  "villagerManip.reset.mobHurt": "Mob Hurt",
+  "villagerManip.reset.day": "Daytime",
+  "villagerManip.reset.swim": "Swum",
+  "villagerManip.reset.potion": "Potion Effect Active",
+  "villagerManip.reset.itemEquipped": "Item Equipped",
+  "villagerManip.reset.gossip": "Gossip",
+  "villagerManip.reset.distance": "Too far from villager",
+  "villagerManip.reset.ambient": "Bad Ambient Noise",
+  "villagerManip.reset.no": "Bad Unhappy Noise",
+  "villagerManip.reset.splash": "Bad Splash Noise",
+  "villagerManip.reset.xpOrb": "Bad XP Orb Size",
+  "villagerManip.reset.yes": "Bad Happy Noise",
+  "villagerManip.reset.itemInMainHand": "Item in Main Hand",
+  "villagerManip.reset.invalidBedPosition": "Invalid Bed Position",
+  "villagerManip.reset.invalidCage": "Invalid Villager Cage",
+
+  "villagerManip.help.day": "Help: Villager RNG manipulation can only be done at night-time",
+  "villagerManip.help.setup.prefix": "Help: Trap the villager with trapdoors at their head with a bed's head exactly 3 blocks away in any direction, as seen in ",
+  "villagerManip.help.setup.here": "this",
+  "villagerManip.help.setup.suffix": " image",
```

We don't use this whole prefix/suffix system anymore. Use `%s` for where the link should go

---

what...? the `here` has it's own style stuff, how the hell am I meant to do that with one string then?

---

I didn't say do it in one string

---

To clarify, I mean this:
```suggestion
  "villagerManip.help.setup": "Help: Trap the villager with trapdoors at their head with a bed's head exactly 3 blocks away in any direction, as seen in %s image",
  "villagerManip.help.setup.link": "this",
```

---

yeah, but then `this` doesn't have an underline and the link isn't limited to `this`, is that ok?

---

No it does work

---

style applies to `%s` inside of a translate?

---

Yes, you can pass text components as `%s` arguments and they will format properly

---

epic